packages/types/src/chains.ts

```diff
@@ -218,4 +218,75 @@ export interface Chain {
   };
 }
 
+export interface MemoKeys {
+  $schema?: string;
+  memo_keys: {
+    key: string;
+    description: string;
+    git_repo: string;
+    memo: {
+      [k: string]: unknown;
+    };
+    [k: string]: unknown;
+  }[];
+}
+
+export interface ChainVersions {
```

Comment:
Here too. What's the usage?

Reply:
Yep I just generated it using chain registry schemas, actually we can also generate this data they have some json for that.

I didn't add it now because they're new data and idk if you wanna add them inside this package